canvas_sdk/v1/data/staff.py

```diff
@@ -46,10 +46,9 @@ def __str__(self) -> str:
     cultural_ethnicity_terms = ArrayField(models.CharField())
     last_known_timezone = TimeZoneField(null=True)
     active = models.BooleanField()
-    # TODO - uncomment when PracticeLocation field is developed
-    # primary_practice_location = models.ForeignKey(
-    #     'v1.PracticeLocation', on_delete=models.DO_NOTHING, null=True
-    # )
+    primary_practice_location = models.ForeignKey(
+        "v1.PracticeLocation", on_delete=models.DO_NOTHING, null=True
+    )
```

Comment:
This change is unrelated, but I was in the neighborhood. Addresses https://github.com/canvas-medical/canvas-plugins/issues/474